src/index.js

```diff
@@ -16,9 +26,32 @@ const menu = {
 const table = ['Mesa 1', 'Mesa 2', 'Mesa 3', 'Mesa 4', 'Mesa 5'];
 
 const waiter = () => {
-  orders(6000, menu.hamburger, table[3])
-    .then((res) => console.log(res))
-    .catch((err) => console.error(err));
+  orders(randomTime(1000,8000) , menu.hamburger, table[3])
+    .then((resolve) => console.log(resolve))
+    .catch((reject) => console.error(reject));
+};
+
+const waiter2 = () => {
+  orders(randomTime(1000,8000) , menu.hotdog, table[0])
```

Comment:
Lo ideal es establecer 2 variables

```suggestion
  orders(randomTime(MIN, MAX) , menu.hotdog, table[0])
```